section4/exercises/burrito.rb

```diff
@@ -11,9 +11,19 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def change_info(protein, base, toppings)
+    @protein  = protein
+    @base     = base
+    @toppings = toppings
+  end
```

Comment:
In the directions we are asking you to create 3 different methods and name them `add_topping`, `remove_topping`, and `change_protein`